api/src/features/todo/use_cases/delete_todo_by_id.py

```diff
@@ -6,15 +6,17 @@
 )
 
 
-class DeleteTodoByIdResponse(BaseModel):
+class DeleteTodoByIdResponseModel(BaseModel):
     success: bool = Field(...)
 
 
-def delete_todo_use_case(id: str, user_id: str, todo_repository: TodoRepositoryInterface) -> DeleteTodoByIdResponse:
+def delete_todo_use_case(
+    id: str, user_id: str, todo_repository: TodoRepositoryInterface
+) -> DeleteTodoByIdResponseModel:
     todo_item = todo_repository.get(id)
     if todo_item is None:
         raise NotFoundException
     if todo_item.user_id != user_id:
         raise MissingPrivilegeException
     todo_repository.delete(id)
-    return DeleteTodoByIdResponse(success=True)
+    return DeleteTodoByIdResponseModel(success=True)
```

Comment:
DIdn't you say this shouldn't have any response model? Can just return a 204 (no idea how that looks in FastAPI).